Sources/Commands/Snippets/Card.swift

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
I wonder if some of the UI for snippets should be in a distinct module from `Commands`. I could see that this would be something we may want to expose via libSwiftPM at some point.

Reply:
I was thinking the same thing actually. Before committing to that, what do you say we keep it here for now while we refine it some more? I think it will probably change a bit in the coming months.

Reply:
Yep, I think that's fine.